scripts/penguin_wake_eval_matrix.sh

```
@@ -79,7 +93,11 @@ run_one() {
   fi
 }
 
-echo "Running $SAMPLES sample(s) per model, up to $MAX_PARALLEL at a time."
+# Printed because the silent version fooled the person who wrote this fix: a
+# run covering one scenario looks exactly like a run covering all of them.
+echo "Models:    $(echo $MODELS | tr '\n' ' ')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)